keyboards/ploopyco/trackball_nano/info.json

```
@@ -0,0 +1,6 @@
+{
+    "keyboard_name": "Ploopy Trackball Nano",
+    "url": "www.ploopy.co",
+    "maintainer": "ploopyco",
+    "manufacturer": "Ploopy Corporation",
+}
```

Horrible hack, but it gets past lint. 
```suggestion
     "layouts": {
        "LAYOUT": {
            "layout": [
                {"x":0, "y":0}
            ]
        }
    }
}
```